### PR TITLE
adding gradio app

### PR DESCRIPTION
**Pull request by sidjain3ai**

### CODE REVIEW

**Comment by sidjain3ai** [comment]
Screenshot 2023-01-31 at 11 51 22 AM

**Comment by sidjain3ai** [comment]
Screenshot 2023-01-31 at 1 30 01 PM

File: SentimentAnalysis.ipynb [hunk 1]
```diff
@@ -21,7 +21,7 @@
    "source": [
     "!pip3 install thirdai --upgrade\n",
     "!pip3 install datasets==2.6.2 \n",
-    "\n",
+    "!pip3 install gradio\n",
```
Comment (vihan29) [inline SentimentAnalysis.ipynb:24]: Could we add a markdown cell explaining introducing the gradio demo (in both notebooks)?

**Review by vihan29 — APPROVED** [review]
This is very cool! Please clear the output of all cells before merging.

**Pull request merged**